src/prices/provider.ts

```diff
@@ -0,0 +1,184 @@
+import { Logger } from "pino";
+import { CHAINS } from "../config.js";
+import { Price, readPricesFile } from "./common.js";
+
+interface PriceProviderConfig {
+  updateEvery?: number; // XXX hint at time type: secs? msecs?
```

Comment:
this would be great - i've seen a `ms` suffix used in the wild, so this would become `updateEveryMs`.